packages/progress-bar/src/ProgressBar/ProgressBar.styles.ts

```diff
@@ -109,7 +105,16 @@ export const getHeaderIconStyles = ({
   margin-bottom: ${spacingToken[50]}px; // align icon with text baseline
   color: ${disabled
     ? colorToken[theme].icon.disabled.default
-    : progressBarColorStyles[theme][color].icon};
+    : barColorStyles[theme][color].icon};
+`;
+
+export const getAnimatedTextStyles = (isNewDescription: boolean) => css`
+  ${isNewDescription &&
```

Comment:
I won't block on this, but we tend to use `cx` for conditional classes. So, for example, instead of checking `isNewDescription &&` and generating conditional styles inside this style obj, we always define the styles in the obj and conditionally compose the styles with `cx`. This way it is clearer and explicit about what styles are applied.

Here's an example from [ContextDrawer](https://github.com/mongodb/leafygreen-ui/blob/main/packages/context-drawer/src/ContextDrawer/ContextDrawer.styles.ts#L73-L92)

So this would be:
```js
className={cx({[animatedTextStyle]: isNewDescription})}
```